### PR TITLE
WIP: FUNFUNZ-90: Created select, checkbox, radio, checkbox-group, text, nu…

### PR DESCRIPTION
**Pull request by miriampereira**
…mber, password, switch inputs

### CODE REVIEW

File: src/components/helpers/serialize.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export function serialize(value: any): string {
```
Comment (joaogsleite) [inline src/components/helpers/serialize.ts:1]: Does it make more sense to have these utils/helpers outside `components/` folder?

File: src/style/grid.scss [hunk 2]
```diff
@@ -0,0 +1,73 @@
+@import './mixins.scss';
+
+// Responsive grid system
+.columns {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: inherit;
+
+  &.columnsnowrap {
```
Comment (joaogsleite) [inline src/style/grid.scss:9]: Can we use camelCase or - for better readability of class names?

Comment (joaogsleite) [reply src/style/grid.scss:9]: the same for other col class names

File: src/components/input/index.tsx [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+    case 'select':
+      return (
+        <select
+          disabled={disabled}
+          onChange={onChange}
+          name={name}
+          value={value}
+        >
+          {options &&
+            options.map((option) => (
```
Comment (joaogsleite) [inline src/components/input/index.tsx]: Each child in a list should have a unique "key" prop

**Review by joaogsleite — APPROVED** [review]

**Review by beajsilva — APPROVED** [review]

**Pull request merged**